inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/JoinRawCalls.json.tmpl

```diff
@@ -0,0 +1,28 @@
+{
+  "JoinRawCalls.gatk_docker": "${workspace.gatk_docker}",
+  "JoinRawCalls.sv_base_mini_docker": "${workspace.sv_base_mini_docker}",
+  "JoinRawCalls.sv_pipeline_docker": "${workspace.sv_pipeline_docker}",
+
+  "JoinRawCalls.clustered_depth_vcfs" : "${this.clustered_depth_vcf}",
+  "JoinRawCalls.clustered_depth_vcf_indexes" : "${this.clustered_depth_vcf_index}",
+
+  "JoinRawCalls.clustered_manta_vcfs" : "${this.clustered_manta_vcf}",
+  "JoinRawCalls.clustered_manta_vcf_indexes" : "${this.clustered_manta_vcf_index}",
+
+  "JoinRawCalls.clustered_wham_vcfs" : "${this.clustered_wham_vcf}",
+  "JoinRawCalls.clustered_wham_vcf_indexes" : "${this.clustered_wham_vcf_index}",
+
+  "JoinRawCalls.clustered_melt_vcfs" : "${this.clustered_melt_vcf}",
+  "JoinRawCalls.clustered_melt_vcf_indexes" : "${this.clustered_melt_vcf_index}",
```

Comment:
Good catch. This looks consistent with the AoU workspace and I've changed the default to Scramble even though that won't be default behavior until #722 goes in. I think this has been running with empty Scramble inputs so that shouldn't be a problem.